agents/dns.py

```diff
@@ -0,0 +1,40 @@
+import logging
+import subprocess
+
+from .agent import ServerAgent
+
+
+class DNSAgent(ServerAgent):
+    server_name = 'dns'
+    port = 53
+
+    def __init__(self, smtp_processes=None):
+        super(DNSAgent, self).__init__()
+
+        self.processes = smtp_processes or ['named', 'bind9']
```

Comment:
I'm sure it's just a typo - let's rename `smtp_processes` to `dns_processes`. However, if we're moving this method to the parent class (which we are), this is not relevant.